rest/server_context.go

```diff
@@ -574,9 +575,9 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 
 	// Create default users & roles:
 	if err := sc.installPrincipals(dbcontext, config.Roles, "role"); err != nil {
-		return nil, fmt.Errorf("Error calling installPrincipals with config.Roles: %v", err)
+		return nil, err
 	} else if err := sc.installPrincipals(dbcontext, config.Users, "user"); err != nil {
-		return nil, fmt.Errorf("Error calling installPrincipals with config.Users: %v", err)
+		return nil, err
```

Comment:
In this case, it feels tricky to differentiate between stack traces without the additional information (roles vs. users).  We'd have to depend on line numbers, which probably means going and looking up the line numbers for the given release.

Reply:
Used double-wrap here since it can add valuable context at this stack frame